.github/workflows/release.yml

```diff
@@ -53,6 +27,18 @@ jobs:
         run: |
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      - name: Tests and Package
+        run: earthly +pre-commit
+      - name: Has filesystem changed ?
+        id: changed-files
+        shell: bash
+        run: |
+          hasChanged=$(git status --porcelain) 
+          if (( $(echo ${#hasChanged}) != 0 )); then
+            git status
+            echo "There are changes in the repository"
+            exit 1
+          fi
```

Comment:
Pre-commit, Test filesystem, then release

Reply:
I would have loved to have something directly in the release target but it's harder